packages/core/src/app/order/OrderSummary.tsx

```diff
@@ -41,6 +119,28 @@ const OrderSummary: FunctionComponent<OrderSummaryProps & OrderSummarySubtotalsP
                 {additionalLineItems}
             </OrderSummarySection>
 
+            {loyaltiPoints?.balance?.available && (
+              <OrderSummarySection>
+                <div className='reward-section'>
+                  <h3 className="loyalty-discount optimizedCheckout-headingSecondary">
+                    Loyalty Discount
+                  </h3>
+                  <div>
+                    <span className='cart-priceItem-value'>{loyaltiPoints?.balance?.available}</span> off available.
```

Comment:
Have we previously discussed the translations on this page?
We probably won't need them on release, but it could be a problem in the future

Reply:
No I haven't discussed, but yeah I can add this in translation file

Reply:
we have translation files in custom-checkout, but ignoring for now